### PR TITLE
notifyRedemptionSignatureTimedOut extracted out of IDeposit

### PR DESCRIPTION
**Pull request by pdyraga**
Depends on: #123 

`IDeposit` is declared in `RiskManagerV1.sol `and should contain only a subset
of tBTC Deposit functions that are used by tBTC v1 Risk Manager.

`notifyRedemptionSignatureTimedOut` is never called by tBTC v1 Risk
Manager and is used only by system tests to transition tBTC `Deposit`
contract into liquidation state.

`notifyRedemptionSignatureTimedOut` has been extracted to a separate
interface with a subset of functions used by system and unit tests.

### CODE REVIEW

File: contracts/test/DepositStub.sol [hunk 1]
```diff
@@ -6,6 +6,13 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 import "../RiskManagerV1.sol";
 
+/// @dev tBTC v1 deposit interace with a subset of functions used by unit and
```
Comment (tomaszslabon) [inline contracts/test/DepositStub.sol:9]: Typo interace -> interface

File: contracts/test/DepositStub.sol [hunk 2]
```diff
@@ -6,6 +6,13 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
...
+///      system tests. For more information about tBTC Deposit, please see:
+///      https://github.com/keep-network/tbtc/blob/solidity/v1.1.0/solidity/contracts/deposit/Deposit.sol
+interface ITBTCDeposit is IDeposit {
```
Comment (tomaszslabon) [inline contracts/test/DepositStub.sol:12]: Shouldn't `DepositStub` contract declared below inherit from `ITBTCDeposit`?
It has `notifyRedemptionSignatureTimedOut` function.

File: contracts/test/DepositStub.sol (hunk 1 above)
Comment (pdyraga) [reply contracts/test/DepositStub.sol]: https://github.com/keep-network/coverage-pools/pull/124/commits/fb8e4ad1cc1efae2b923e7e53bf8031bf0ab34ae

File: contracts/test/DepositStub.sol (hunk 2 above)
Comment (pdyraga) [reply contracts/test/DepositStub.sol:12]: https://github.com/keep-network/coverage-pools/pull/124/commits/fb8e4ad1cc1efae2b923e7e53bf8031bf0ab34ae

**Review by tomaszslabon — APPROVED** [review]

**Pull request merged**